Parser/pegen/pegen.h

```diff
@@ -128,14 +128,33 @@ expr_ty _PyPegen_number_token(Parser *p);
 void *_PyPegen_string_token(Parser *p);
 const char *_PyPegen_get_expr_name(expr_ty);
 void *_PyPegen_raise_error(Parser *p, PyObject *errtype, int with_col_number, const char *errmsg, ...);
+void *_PyPegen_raise_error_known_location(Parser *p, PyObject *errtype,
+                                   int with_col_number,
+                                   int lineno, int col_offset,
```

Comment:
Can't we change the interface to remove the silly `with_col_number` flag and set `col_offset` to -1 instead? (You still would have to check for it and replace it with None, but it makes the call a bit less unwieldy.)